services/core-web/common/utils/helpers.ts

```diff
@@ -593,7 +590,7 @@ export const getHighestConsequence = (tsf) => {
 
   const highestRankedDam = tsf.dams.reduce((prev, current) =>
     CONSEQUENCE_CLASSIFICATION_RANK_HASH[prev.consequence_classification] >
-      CONSEQUENCE_CLASSIFICATION_RANK_HASH[current.consequence_classification]
+    CONSEQUENCE_CLASSIFICATION_RANK_HASH[current.consequence_classification]
```

Comment:
Just a note on this helpers.ts file (I know it's not ready for review and haven't really looked at the rest!)-

There is a newer version of the file in `services/common/src/redux/utils`. Lately, we tend to update _that_ version, and just update the reference in whatever's importing from it. Then you don't need to make changes in both core/ms versions. We are planning to do a big ticket at some point to update _all_ the references, and a change like this might get missed!

Reply:
Oh wait I see now that this is just a whitespace difference. Don't worry about it!

Reply:
Yea, my vs studio formatter and the git pre-hook formatter are competing i think